.github/workflows/openqa_fullstack.yml

```
@@ -0,0 +1,21 @@
+---
+name: openQA Fullstack
+# yamllint disable-line rule:truthy
+on: [push, pull_request]
+jobs:
+  fullstack:
+    runs-on: ubuntu-latest
+    container:
+      image: registry.opensuse.org/devel/openqa/ci/containers/base:latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Make project folder user-writable
+        run: sudo chown -R $USER ../../os-autoinst
+      - name: Clone openQA
+        run: git clone https://github.com/os-autoinst/openQA.git ../openQA
+      - name: Install dependencies
+        run: (cd ../openQA; bash -x tools/ci/build_cache.sh)
+      - name: Build os-autoinst
+        run: bash -x tools/ci/build_autoinst.sh
```

Instead of a new script file it can be just as simple as

```suggestion
        run: make symlinks
```

and not needing the build script at all